### ISSUE TITLE
feat(a11y): please look into providing an accessible experience

### ISSUE THREAD

**Issue by Splaktar** [stackblitz/core#731]
Hello!

Thank you for the great work!

As part of AngularJS Material, we currently host all of our demos/bug repros/etc on CodePen. I've been wondering if there was a reason for me to switch it all over to Stackblitz. I thought that I might have found it today, but alas... no luck.

I had a contributor who was unable to provide a bug reproduction because CodePen's editor wasn't accessible enough for a blind person to use. I suggested that he give Stackblitz a shot and provided him a link. His feedback is captured [here](https://github.com/angular/material/issues/11485#issuecomment-430101680) but the basic feedback was that the Stackblitz editor was even less accessible than CodePen's.

I know that you guys have a lot of demands and different things going on, but I just wanted to make sure that accessibility is on your roadmap and provide a concrete case where a great developer (from Mozilla) was unable to use your product due to lacking accessibility support.

As one of Stackblitz's goals is to make coding and learning coding more accessible to a wider range of developers, this seems to fall directly in line with your mission 😄
